tactical-microgrid-standard/controller/Controller.mpc

```diff
@@ -0,0 +1,57 @@
+project(*Commands): dcps_idl_only_lib, opendds_cxx11 {
```

Comment:
Consider putting each project in its own directory.  It's easier to understand how the code fits together that way.